data_model/src/events/data/filters.rs

```diff
@@ -458,6 +458,10 @@ mod account {
         ByPermissionAdded,
         /// Filter by PermissionRemoved event
         ByPermissionRemoved,
+        /// Filter by RoleRevoked event
+        ByRoleRevoked,
+        /// Filter by RoleGranted event
+        ByRoleGranted,
         /// Filter by MetadataInserted event
         ByMetadataInserted,
         /// Filter by MetadataRemoved event
```

Comment:
```suggestion
        ByPermissionAdded,
        /// Filter by `PermissionRemoved` event
        ByPermissionRemoved,
        /// Filter by `RoleRevoked` event
        ByRoleRevoked,
        /// Filter by `RoleGranted` event
        ByRoleGranted,
        /// Filter by `MetadataInserted` event
        ByMetadataInserted,
        /// Filter by `MetadataRemoved` event
```

plus the same change for the lines above these